### DISCUSSION TITLE
为什么组件都要加个-啊。

### DISCUSSION THREAD

**Discussion by FalconWu2017** [ofajs/discussion#14] [General]
感觉没什么理由，只是为了表示他是个组件么？

**Comment by kirakiray** [comment]
😂 因为 ofa.js 是基于浏览器原生的 [Web Components API](https://developer.mozilla.org/zh-CN/docs/Web/API/Web_components) 开发的，必须带有 `-` 分隔，应该是为了和原生标签进行区分。
